config/initializers/apipie.rb

```diff
@@ -20,6 +20,7 @@
     :providers => ComputeResource.providers.join(', '),
     :default_nic_type => InterfaceTypeMapper::DEFAULT_TYPE.humanized_name.downcase,
     :template_kinds => -> { TemplateKind.scoped.map(&:name).join(", ") },
+    :provision_methods => Host::Managed.provision_methods.map { |method, friendly_name| "#{method} (#{friendly_name})" }.join(', '),
```

Comment:
friendly_name here won't get translated into the per-language API docs.  You could leave it out or change this to a lambda so it's run on the fly, so it can be translated correctly: `-> Host::Managed.provision_methods.map { |method, friendly_name| "#{method} (#{_(friendly_name)})" }.join(', '),`